vnext/Microsoft.ReactNative/Modules/CreateModules.cpp

```diff
@@ -28,9 +27,7 @@ std::shared_ptr<IWebSocketResource> IWebSocketResource::Make(std::string &&urlSt
 
 std::unique_ptr<facebook::xplat::module::CxxModule> CreateWebSocketModule(
     Mso::CntPtr<Mso::React::IReactContext> &&context) noexcept {
```

Comment:
Can we remove the param for React context if no longer needed?

Consider removing this function entirely and just inlining the calls to make_unique if we don't need a factory function.